microsoft/testsuites/power/common.py

```diff
@@ -37,6 +37,8 @@ def verify_hibernation(
     log: Logger,
     throw_error: bool = True,
 ) -> None:
+    if isinstance(node.os, Redhat):
+        node.os.grow_os_partition_size()
```

Comment:
add comments for why need grow os size for rhel lvm images

Reply:
Added